eland/groupby.py

```diff
@@ -164,3 +169,14 @@ def aggregate(self, func: List[str], numeric_only: bool = False) -> "pd.DataFram
         )
 
     agg = aggregate
+
+    def count(self) -> "pd.DataFrame":
+        """
+        Used to groupby and count
+
+        Returns
+        -------
+            A Pandas DataFrame
+
+        """
+        return self._query_compiler.count_groupby(by=self._by, is_dataframe_agg=False)
```

Comment:
Instead of adding a new function `count_groupby()` can we instead use `aggs_groupby()` since it seems that `count_groupby()` simply calls that function?

Reply:
Yeah!, I totally missed that.